conf/application.conf

```diff
@@ -64,6 +64,10 @@ mongodb {
   uri = "mongodb://localhost:27017/emcs-tfe-crdl-reference-data"
 }
 
+internal-auth {
+  token = "crdl-cache-token"
```

Comment:
This token needs setting up the first time the service is called locally.

This can be done using the [test-only token endpoint](https://github.com/hmrc/internal-auth/tree/main?tab=readme-ov-file#endpoints) of internal-auth.